src/core/Tooltip/index.tsx

```diff
@@ -77,13 +78,42 @@ export default forwardRef(function Tooltip(
 
   const leaveDelay = inverted || sdsStyle === "dark" ? 0 : 500;
 
+  const [position, setPosition] = React.useState({ x: 0, y: 0 });
+
   return (
     <RawTooltip
       classes={{ arrow, tooltip }}
       leaveDelay={leaveDelay}
       interactive
       title={content}
       ref={ref}
+      onMouseMove={
+        followCursor
+          ? (e) => setPosition({ x: e.pageX, y: e.pageY })
```

Comment:
We probably just need to keep an eye on `onMouseMove()`, since it is called without throttling, so we're setting state as fast as every 6ms. 

For perf, we could throttle the set state to be roughly every 16ms to match 60fps, but that would require adding `lodash/throttle` to SDS as a dependency, I'm not sure it's worth it right now given the tooltip rendering seems cheap

But in the future, if products do see perf issue with this, we can check if throttling will help!